website/docs/r/virtual_network_gateway_connection.html.markdown

```diff
@@ -245,6 +245,12 @@ The following arguments are supported:
     Site-to-Site or VNet-to-VNet connection is created whereas ExpressRoute
     connections do not need a shared key.
 
+* `connection_protocol` - (Optional) The IKE protocol version to use. Possible
+    values are `IKEv1` and `IKEv2`. Only valid for IPSec connections on virtual
+    network gateways with SKU `VpnGw1`, `VpnGw2`, `VpnGw3`, `VpnGw1AZ`, `VpnGw2AZ`
+    or `VpnGw3AZ`. Changing the protocol will force a new connection to be created. 
+    Defaults to `IKEv2`. 
```

Comment:
There's no default set above? As such should we set this to:

```suggestion
```

Reply:
It is implicit set to IKEv2 by the API if the value is emtpy. Hence the `if` check above

Reply:
Also I wouldn't want to introduce it as a default value on the provider, as it would cause all existing connections to be recreated. But as the default if not set is IKEv2 I have no better idea on how to document this...